common/predictive-text/worker/correction/distance-modeler.ts

```diff
@@ -281,6 +281,9 @@ namespace correction {
       this.buildQueueSpaceComparator();
       this.selectionQueue = new models.PriorityQueue<SearchSpaceTier>(this.QUEUE_SPACE_COMPARATOR);
       this.rootNode = new SearchNode(traversalRoot);
+      if(!traversalRoot) {
+        console.error();
```

Comment:
I'm pretty sure this was deleted in #3575 but this should really THROW an error instead of print an empty error message! 😂

Reply:
Yeah... kinda realized it at the time.  Kinda figured I'd just address it with any other review concerns if requested, while making sure it was at least covered _somewhere_.  Had a bit of branch re-org madness going on that day.